maestro/onboarding.yaml

```diff
@@ -0,0 +1,15 @@
+appId: im.status.ethereum.debug
+---
+- tapOn: "I’m new to Status"
+- tapOn: "small-option-card"
+- tapOn: "Your name"
+- inputText: "DeviceA"
```

Comment:
we could take inspiration from these examples to generate random usernames and passwords : https://github.com/mobile-dev-inc/maestro/blob/main/samples/wikipedia-android-advanced/scripts/generateCredentials.js#L2